requirements.txt

```diff
@@ -11,6 +11,10 @@ mosaicml-streaming==0.13.0
 pre-commit
 ruff
 sentencepiece==0.2.1
-torch==2.5.1
+# torch==2.5.1
+# Upgrade torch due to:
+#   ╰─▶ Because mosaicml==0.32.1 depends on torch>=2.6.0,<2.7.1 and you require mosaicml==0.32.1, we can conclude that you require torch>=2.6.0,<2.7.1.
+#       And because you require torch==2.5.1, we can conclude that your requirements are unsatisfiable.
+torch==2.7.0
```

Comment:
I think I saw this inconsistency before in the environment and forgot to fix it.  @yair-schiff / @mariannearr do you say any reason not to use torch 2.7.0 from now on?  Torch 2.5.1 is too old for composer 0.32.1.

Reply:
Let's upgrade